15/umbraco-commerce/how-to-guides/configuring-cart-cleanup.md

```diff
@@ -0,0 +1,44 @@
+---
+description: Learn how to configure a cart cleanup routine.
+---
+
+# Configuring Cart Cleanup
+
+{% hint style="info" %}
+Available from Umbraco Commerce 15.1.0
+{% endhint %}
+
+By default Umbraco Commerce will keep all created carts indefinately. Over time this can obviously become an issue. To assist with this is it possible to configure a cart cleanup routine to delete carts older than a pre-configured time interval.

```

Comment:
```suggestion
By default Umbraco Commerce will keep all created carts indefinately. Over time this can become an issue. To assist with this is it possible to configure a cart cleanup routine to delete carts older than a pre-configured time interval.

```